frontend/utils/api.js

```diff
@@ -18,6 +18,29 @@ const formatGroceryItemAdd = (formData) => {
     }
 }
 
+
+
+export const addPrice = (formData) => {
```

Comment:
There are some duplicate codes in addPrice() and addGroceryItem(). Is it possible to reuse/modularize part of the code?

Reply:
We'll definitely look into reusing that code to avoid duplication